### PR TITLE
E85 Hotswap/Soldered Fixes

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Apparently the Hotswap and the Soldered variants have the Pipe and Enter key mixed up. 

This PR separates the two into different keyboards Hotswap and Soldered. 

Thanks to [RebeccaBy](https://github.com/rebeccaby) for finding the correct positions for enter and pipe for the soldered PCB. Thanks to ants eclipse for pointing it out. 

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/exclusive/e85/hotswap/info.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
+{
+    "keyboard_name": "Exclusive E8.5 TKL",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 18.25,
+    "height": 6.5,
+    "layouts": {
+        "LAYOUT_ansi_standard": {
+            "key_count": 88,
```
Comment (noroadsleft) [inline keyboards/exclusive/e85/hotswap/info.json:9]:
```suggestion
```

File: keyboards/exclusive/e85/hotswap/info.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                {"label":"K54 (B7,D5)", "x":12.5, "y":5.5, "w":1.25},
+                {"label":"K53 (B7,D3)", "x":13.75, "y":5.5, "w":1.25},
+                {"label":"KA6 (F1,D6)", "x":15.25, "y":5.5},
+                {"label":"KA2 (F1,D2)", "x":16.25, "y":5.5},
+                {"label":"KA1 (F1,D1)", "x":17.25, "y":5.5}
+            ]
+        },
+
+        "LAYOUT_ansi_tsangan": {
+            "key_count": 87,
```
Comment (noroadsleft) [inline keyboards/exclusive/e85/hotswap/info.json:103]:
```suggestion
```

File: keyboards/exclusive/e85/hotswap/info.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+                {"label":"K57 (B7,D7)", "x":11, "y":5.5, "w":1.5},
+                {"label":"K54 (B7,D5)", "x":12.5, "y":5.5},
+                {"label":"K53 (B7,D3)", "x":13.5, "y":5.5, "w":1.5},
+                {"label":"KA6 (F1,D6)", "x":15.25, "y":5.5},
+                {"label":"KA2 (F1,D2)", "x":16.25, "y":5.5},
+                {"label":"KA1 (F1,D1)", "x":17.25, "y":5.5}
+            ]
+        }
+    }
+    ,"meta": "https://noroadsleft.github.io/kbf_qmk_converter/"
```
Comment (noroadsleft) [inline keyboards/exclusive/e85/hotswap/info.json:195]:
```suggestion
```

File: keyboards/exclusive/e85/rules.mk [hunk 4]
```diff
@@ -25,8 +18,6 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 NKRO_ENABLE = no            # USB Nkey Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
-MIDI_ENABLE = no            # MIDI support
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
-FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 LTO_ENABLE = yes
```
Comment (noroadsleft) [inline keyboards/exclusive/e85/rules.mk:23]: I have a slight preference for maintaining behaviour for users who are used to typing `make exclusive/e85:keymap` to compile their firmware.

```suggestion
LTO_ENABLE = yes

DEFAULT_FOLDER = exclusive/e85/hotswap
```

File: keyboards/exclusive/e85/config.h [hunk 5]
```diff
@@ -21,11 +21,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID 0x4558 // EX
-#define PRODUCT_ID 0xE851 // E8.5 Version 1
 #define DEVICE_VER 0x0001
 #define MANUFACTURER Exclusive
 #define PRODUCT E8.5 Hotswap PCB
```
Comment (noroadsleft) [inline keyboards/exclusive/e85/config.h:26]:
```suggestion
```

File: keyboards/exclusive/e85/hotswap/config.h [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#define PRODUCT_ID 0xE851 // E8.5 Hotswap
```
Comment (noroadsleft) [inline keyboards/exclusive/e85/hotswap/config.h:17]:
```suggestion
#define PRODUCT_ID 0xE851 // E8.5 Hotswap
#define PRODUCT E8.5 Hotswap PCB
```

File: keyboards/exclusive/e85/soldered/config.h [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#define PRODUCT_ID 0xE852 // E8.5 Soldered
```
Comment (noroadsleft) [inline keyboards/exclusive/e85/soldered/config.h:17]:
```suggestion
#define PRODUCT_ID 0xE852 // E8.5 Soldered
#define PRODUCT E8.5 Soldered PCB
```

**Comment by mechmerlin** [comment]
> `keyboards/exclusive/e85/soldered/keymaps/tsangan/keymap.c` is identical to `keyboards/exclusive/e85/soldered/keymaps/default/keymap.c` - I don't see how both are necessary to keep. Maybe delete `default` and rename `standard` to `default`, or delete `tsangan`? I'm not sure which would be considered the most useful to users of this board.
> 
> Everything else looks good though.

I'll take out the tsangan keymap

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**